.license_check.yaml

```diff
@@ -28,51 +28,13 @@
 # concerned, they can be considered the same.
 
 file_types:
-  - pattern: "*.service"
+  "*.service":
     type: yaml
-  - pattern: "*.spec"
+  "*.spec":
     type: yaml
-  - pattern: "*.sh"
+  "goss-testing/automated/*":
```

Comment:
There are also scripts in:

- `goss-testing/scripts`
- `systemd`

Those have licenses already so should also be included

Reply:
Also under `src`

Reply:
I see that all files under `goss-testing/scripts/` have proper `.sh` extension, so they are covered by base default configuration, which associates `*.sh` with type `shell_or_python`. Same for file under `src/` - they all match `*.py` and thus treated as Python files. For files under `systemd/`, `goss-servers.service` is covered by `*.service` pattern defined above, and `start-goss-servers.sh` is a regular shell script covered by base config. So the only file not covered is `systemd/90-goss-servers.preset`. I'll add `systemd/*.preset` pattern.

Reply:
Done. FYI, there's few files not recognized by base config, and few files missing license header:

    $ docker run -it --rm -v $(pwd):/github/workspace artifactory.algol60.net/csm-docker/stable/license-checker | grep -E 'not (detected|recognized)'
    [INFO] Filename pattern not recognized: ./pyproject.toml
    [INFO] Filename pattern not recognized: ./.shellspec
    [INFO] Filename pattern not recognized: ./goss-testing/dat/goss-servers.cfg
    [WARNING] License is not detected: ./spec/functional/dst-ct-results_spec.sh
    [WARNING] License is not detected: ./.github/dependabot.yml
    [WARNING] License is not detected: ./.github/workflows/merge-strategy.yml